NorthstarDLL/CMakeLists.txt

```diff
@@ -148,12 +151,12 @@ add_library(NorthstarDLL SHARED
             "audio_asm.asm"
             "dllmain.cpp"
             "dllmain.h"
-            "ns_version.h"
-)
+            "ns_version.h")
 
 target_link_libraries(NorthstarDLL PRIVATE
                       minhook
                       libcurl
+					  minizip
```

Comment:
So it seems we actually spaces to indent `CMakeLists.txt`. Good thing were are so consistent in our indentation methods (:

Reply:
This was intentional (and explained in the CMake PR) because of how CMake files are typically aligned, tabs aren't really suitable here

Reply:
Addressed in 6d094c1a84331189a9b7eaf26d3ac18a3eddd184.